src/analyze/analyze-chid.c

```diff
@@ -223,6 +247,71 @@ static int smbios_fields_acquire(char16_t *fields[static _CHID_SMBIOS_FIELDS_MAX
         return 0;
 }
 
+static int edid_parse(sd_device *drm_card, char16_t **ret_panel) {
+        assert(drm_card);
+        assert(ret_panel);
+        int r;
+        const char *edid_content = NULL;
```

Comment:
Why are `edid_content` and `edid_size` initialized? We have the general rule that functions **must** set output parameters if they don't return failure. So `sd_device_get_sysattr_value_full` would unconditionally set those on success.